gno.land/cmd/gnoland/secrets_get.go

```diff
@@ -26,13 +30,10 @@ func newSecretsGetCmd(io commands.IO) *commands.Command {
 	cmd := commands.NewCommand(
 		commands.Metadata{
 			Name:       "get",
-			ShortUsage: "secrets get [flags] [<key>]",
-			ShortHelp:  "shows all Gno secrets present in a common directory",
-			LongHelp: fmt.Sprintf(
-				"shows the validator private key, the node p2p key and the validator's last sign state. "+
-					"If a key is provided, it shows the specified key value. Available keys: %s",
-				getAvailableSecretsKeys(),
-			),
+			ShortUsage: "secrets get <key>",
```

Comment:
```suggestion
			ShortUsage: "secrets get [flags] <key>",
```

Reply:
I noticed other mismatches, so I updated it here:

[4b6b337](https://github.com/gnolang/gno/pull/2393/commits/4b6b3379dca114e7b41502b9c0436e315a01676e)